tools/archetype/binding.test/src/main/resources/archetype-resources/src/test/java/__bindingIdCamelCase__OSGiTest.java

```diff
@@ -51,8 +51,12 @@
     @Before
     public void setUp() {
         registerService(volatileStorageService);
+
         managedThingProvider = getService(ThingProvider.class, ManagedThingProvider.class);
+        assertNotNull("Could not get ManagedThingProvider", managedThingProvider);
```

Comment:
I understand the intention of the explanatory texts, but I personally dislike them because they state the opposite of what should be asserted. They are great if something goes wrong, but they actually make the code harder to read. 
It might not be such of a big issue here, but we have to keep in mind that this code should server as an example - and in that case I think it is leading towards bad style. 

Therefore I suggest to only add the assertions themselves because they indeed are quite helpful for figuring out what went wrong at the ealiest point possible, but drop the explanatory texts.